keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -715,30 +717,25 @@ to describe a selector based policy that applies to a multi-cluster service.
 
 _Optional, but recommended._
 
-MCS aims to align with the existing [service DNS
+The full specification for Multicluster Service DNS is in this KEP's [specification.md](specification.md). MCS aims to align with the existing [service DNS
 spec](https://github.com/kubernetes/dns/blob/master/docs/specification.md). This
-section assumes familiarity with in-cluster Service DNS behavior.
-```
-<<[UNRESOLVED]>>
-Full DNS spec needed prior to Beta graduation following:
-https://github.com/kubernetes/dns/blob/master/docs/specification.md
-<<[UNRESOLVED]>>
-```
-When a `ServiceExport` is created, this will cause a domain name for the
-multi-cluster service to become accessible from within the clusterset. The
-domain name will be `<service>.<ns>.svc.clusterset.local`. 
+section provides an overview of the multicluster DNS specification and its rationale, and assumes familiarity with in-cluster Service DNS behavior.
 
-**ClusterSetIP services:** Requests to this domain name from within an importing
-cluster will resolve to the clusterset IP, which points to endpoints for pods
-within the underlying `Service`(s) across the clusterset.
+In short, when a `ServiceExport` is created, this will cause a domain name for the
+multi-cluster service to become accessible from within the clusterset. The
+domain name will be `<service>.<ns>.svc.clusterset.local`. This domain name operates differently depending on whether the `ServiceExport` refers to a ClusterSetIP or Headless service:
 
-**Headless services:** Within an importing cluster, the clusterset domain name
+  * **ClusterSetIP services:** Requests to this domain name from within an importing
+cluster will resolve to the clusterset IP. Requests to this IP will be load-balanced
```

Comment:
s/load-balanced/spread ?  We don't promise to "balance" the load

Reply:
Fixed in [c4e0549](https://github.com/kubernetes/enhancements/pull/2577/commits/c4e05493b6917d0fe1d8d451367259061eac704a)